tutorials/2d/2d_lights_and_shadows.rst

```diff
@@ -6,238 +6,324 @@
 Introduction
 ------------
 
-This tutorial explains how the 2D lighting works in the
-`lights and shadows <https://github.com/godotengine/godot-demo-projects/tree/master/2d/lights_and_shadows>`_ demo project.
-It begins with a brief description of the resources used in the final demo and then describes how
-to make a scene like the demo step by step.
+By default, 2D scenes in Godot are fully lit up, with no lights and shadows
```

Comment:
Should we be calling the scene "fully lit"? To me that seems confusing as the scene is actually "fully unlit". By comparison to 3D, a 2D scene is equivalent to a 3D scene where all materials are set to "unlit" and lights have no impact. Once you add a light, the pixels touched by the light become "lit"